lib/packages/core/src/currency.ts

```diff
@@ -0,0 +1,50 @@
+/*
+* Copyright 2018 PoC-Consortium
+*/
+
+/*
+* Currency class
+*
+* The Currency class is a mapping of the currency json object provided by coinmarketcap.com's API.
+*/
+export class Currency {
+    public id: string;
+    public availableSupply: number;
+    public currency: string;
+    public lastUpdated: number;
+    public marketCapCur: number;
+	public marketCapUSD: number;
+    public name: string;
+	public percentChange1h: number;
+	public percentChange24h: number;
+	public percentChange7d: number;
+    public priceBTC: number;
+    public priceCur: number;
+	public priceUSD: number;
+    public rank: number;
```

Comment:
Looks like we got some tabs vs spacing issues. Todo: add ts-lint to the pre-commit hook